src/store/app.ts

```diff
@@ -0,0 +1,7 @@
+import { networkConfigs } from 'config';
+import { atomWithStorage } from 'jotai/utils';
```

Comment:
I thought we were moving away from Jotai. This WithStorage utility is pretty handy tho

Reply:
I'd prefer not to and don't recall that decision, I think it will be useful going forward but we can discuss more.